wire-ios-sync-engine/Source/UserSession/UserSession.swift

```diff
@@ -213,7 +213,7 @@ public protocol UserSession: AnyObject {
     var networkState: ZMNetworkState { get }
 
     var getUserClientFingerprint: GetUserClientFingerprintUseCaseProtocol { get }
-    var isSelfUserProteusVerifiedUseCase: IsSelfUserProteusVerifiedUseCaseProtocol { get }
+    var isUserE2EICertifiedUseCase: IsUserE2EICertifiedUseCaseProtocol { get }
```

Comment:
Do we still need both `isUserE2EICertifiedUseCase` and `isUserE2EICertifiedUseCase` (the next line)?

Reply:
Yes, that is intended.